src/main/java/com/mcmoddev/orespawn/data/Constants.java

```diff
@@ -0,0 +1,10 @@
+package com.mcmoddev.orespawn.data;
+
+public class Constants {
+	public static final String MODID = "orespawn";
+	public static final String NAME = "MMD OreSpawn";
+	public static final String VERSION = "3.0.0.0";

```

Comment:
This should be addressed in the current set of changes I pushed to my working tree.